Sources/ToolchainRegistry/ToolchainRegistry.swift

```diff
@@ -150,19 +147,20 @@ package final actor ToolchainRegistry {
   /// * (Darwin) `[~]/Library/Developer/Toolchains`
   /// * `env SOURCEKIT_PATH, PATH`
   package init(
-    installPath: AbsolutePath? = nil,
+    installPath: URL? = nil,
     environmentVariables: [ProcessEnvironmentKey] = ["SOURCEKIT_TOOLCHAIN_PATH"],
-    xcodes: [AbsolutePath] = [_currentXcodeDeveloperPath].compactMap({ $0 }),
-    darwinToolchainOverride: String? = ProcessEnv.block["TOOLCHAINS"],
-    _ fileSystem: FileSystem = localFileSystem
+    xcodes: [URL] = [_currentXcodeDeveloperPath].compactMap({ $0 }),
+    libraryDirectories: [URL] = FileManager.default.urls(for: .libraryDirectory, in: .allDomainsMask),
+    pathEnvironmentVariables: [ProcessEnvironmentKey] = ["SOURCEKIT_PATH", "PATH", "Path"],
```

Comment:
Was `ProcessEnvironmentKey` the modification that I had done to enable case insensitive environment variables on Windows? If so, aren't `Path` and `PATH` handled the same already?

Reply:
Oh, I didn’t know that was the purpose of `ProcessEnvironmentKey`. Guess we don’t need `Path` then.